sumo_mgmt.py

```diff
@@ -46,6 +46,10 @@
 parser.add_argument('-url', metavar='', type=str, nargs=1, help='URL for API call')
 parser.add_argument('-filter', metavar='', type=str, nargs=1, help='(OPTIONAL) filter list of all collectors by given type and condition')
 parser.add_argument('-listVersions', action='store_true', help='list the versions of a given set of collectors')
+parser.add_argument('-getsources', action='store_true', help='gets the sources for given set of collectors')
+parser.add_argument('-changesource', metavar='', type=str, nargs=2, help='(OPTIONAL) changes the source for given set of collectors . Takes a path and a sourcename')
```

Comment:
instead of `-changesource` can we name this `-updateSource`? I think this is more descriptive of the fact it's doing a PUT.  We should also call out it will update the sources with the given name.

Reply:
I don't think `(OPTIONAL)` prefix is needed here since this is a command like `-listVersions` or `-addSource`, not an optional filter.